pwnlib/tubes/ssh.py

```diff
@@ -30,7 +30,10 @@
 # Kill the warning line:
 # No handlers could be found for logger "paramiko.transport"
 paramiko_log = logging.getLogger("paramiko.transport")
-h = logging.StreamHandler(open('/dev/null','w+'))
+if sys.platform != 'win32':
+    h = logging.StreamHandler(open('/dev/null','w+'))
```

Comment:
This can be done using [`os.devnull`](https://docs.python.org/3/library/os.html#os.devnull). Please use it instead.